pageserver/src/virtual_file/open_options.rs

```diff
@@ -127,7 +127,7 @@ impl OpenOptions {
 }
 
 impl std::os::unix::prelude::OpenOptionsExt for OpenOptions {
-    fn mode(&mut self, mode: u32) -> &mut OpenOptions {
+    fn mode(&mut self, mode: u32) -> &mut Self {
```

Comment:
Stop implementing std::os::unix::prelude::OpenOptionsExt, just move to the main impl block

The same for custom_flags